Python/passthecert.py

```diff
@@ -564,13 +612,20 @@ def generateComputerName(self):
             # An EXTERNAL bind is not allowed, and the bind will be rejected with an error."
             # Using bind() function will raise an error, we just have to open() the connection
             ldapConn.open()
-        
-        if options.action in ('forcePWDchange'):
-            manage = ManageUserPWD(ldapConn, options)
-            manage.changePW()
-            sys.exit(1)
 
-        if options.action in ('add_computer','del_computer','modify_computer', 'whoami'):
+        if options.action in ('modify_user'):
+            if options.target is None:
+                logging.critical('-target is required !')
+                sys.exit(1)
+            manage = ManageUser(ldapConn, options)
+            if options.elevate:
+                manage.elevate()
+            elif options.new_pass is not None:
+                manage.changePWD(options.new_pass)
+            else:
+                logging.critical('User modification option (-elevate|-changePWD) needed!')
```

Comment:
Can you change it to:
`ogging.critical('User modification option (-elevate|-new-pass) needed!')`